yabt/builders/filegroup_test.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+
+from yabt.buildcontext import BuildContext
+from yabt.graph import populate_targets_graph
+
+slow = pytest.mark.skipif(not pytest.config.getoption('--with-slow'),
+                          reason='need --with-slow option to run')
+
+
+@slow
+@pytest.mark.usefixtures('in_cpp_project')
+def test_file_group_dep(basic_conf):
+    build_context = BuildContext(basic_conf)
+    basic_conf.targets = ['hello:hello-with-data']
+    populate_targets_graph(build_context, basic_conf)
+    build_context.build_graph(run_tests=True)
```

Comment:
since the thing you're testing here is that the data dir was copied correctly, I suggest:

1. making the data dir a little more interesting (e.g. 1-2 sub-dirs with a couple of files)
2. add a test assertion here that verifies the dir was indeed copies as expected (e.g. run a `tree` command inside the resulting docker image and compare the output to the expected dir tree)

[here's an example](https://github.com/resonai/ybt/blob/2d92a348cb709e46f02f027f923d94932c25a6c6/yabt/docker_test.py#L40-L58) of similar concept in another test